src/App.css

```diff
@@ -1,8 +1,8 @@
-.App {
+np.App {
```

Comment:
This won't work. If you don't need this anymore, yo ucan just delete these three lines.